scripts/geoid_tool.py

```diff
@@ -0,0 +1,254 @@
+"""
```

Comment:
It's a good (and common) practice to start an executable script with this
```
#!/usr/bin/env python
```

Some example reading:
https://realpython.com/python-shebang/